### DISCUSSION TITLE
How to test Select component using testing-library-react?

### DISCUSSION THREAD

**Discussion by gabrielpenhazup** [radix-ui/primitives#1625] [Help]
I'm trying to use testing-library-react to test the Select component, however, I can't find the options, I can't check if they are rendering. I can't click on options

![image](https://user-images.githubusercontent.com/82623871/185613990-c6c5a0ff-16ca-490c-bbf1-f1f56611debf.png)

**Comment by mikedidthis** [comment]
I would try `screen.getByRole("option", { name: "Option 2" }).click()`.
